.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+## 🚨 Proposed changes
```

Comment:
we can add also issue template?

Reply:
yes, we can. But I will do it in another PR.